website/docs/api/tokenizer.md

```diff
@@ -18,11 +18,14 @@ Create a `Tokenizer`, to create `Doc` objects given unicode text.
 > from spacy.tokenizer import Tokenizer
 > from spacy.lang.en import English
 > nlp = English()
+> # Create a blank Tokenizer with just the English vocab
 > tokenizer = Tokenizer(nlp.vocab)
 >
 > # Construction 2
 > from spacy.lang.en import English
 > nlp = English()
+> # Create a Tokenizer with the default settings for English
+> # including matching rules and exceptions
```

Comment:
```suggestion
> # including punctuation rules and exceptions
```

I think this is a bit more straightforward – othwerwise, a user may think that it's actually rules for the `Matcher` and get confused.

Reply:
Hehe, "punctuation" was what I wrote in the first place but after another look at the code I changed it. But you're right it most likely would have lead to confusion.